src/main/java/org/apache/commons/text/similarity/FuzzyScore.java

```diff
@@ -47,7 +72,8 @@
      */
     @Override
     public Integer compare(CharSequence term, CharSequence query) {
-        return compare(term, query, Locale.getDefault());
+        Locale locale = this.locale != null ? this.locale : Locale.getDefault();
```

Comment:
I providing a default for the locale field a good idea at all? I don't think so. I think the locale parameter has to be mandatory.